### ISSUE TITLE
Links in the docs point to notebooks that are not existing anymore

### ISSUE THREAD

**Issue by afisc** [GLIMS-RGI/rgitools#54]
Certain links on this [**docpage**](https://rgitools.readthedocs.io/en/latest/dems.html) are not working:

- The [**Jupyter notebook**](https://github.com/OGGM/oggm-edu-notebooks/blob/master/oggm-tuto/dem_comparison.ipynb) and [**this link**](https://mybinder.org/v2/gh/OGGM/binder/master?urlpath=git-pull?repo=https://github.com/OGGM/oggm-edu-notebooks%26amp%3Bbranch=master%26amp%3Burlpath=lab/tree/oggm-edu-notebooks//oggm-tuto/dem_comparison.ipynb%3Fautodecode) in the [Examples](https://rgitools.readthedocs.io/en/latest/dems.html#examples) section are pointing to outdated sources
-  Also the [**tutorial**](https://github.com/OGGM/oggm-edu-notebooks/blob/master/oggm-tuto/dem_sources.ipynb) and [**interactive version**](https://mybinder.org/v2/gh/OGGM/binder/master?urlpath=git-pull?repo=https://github.com/OGGM/oggm-edu-notebooks%26amp%3Bbranch=master%26amp%3Burlpath=lab/tree/oggm-edu-notebooks//oggm-tuto/dem_sources.ipynb%3Fautodecode) links in the [Code availability](https://rgitools.readthedocs.io/en/latest/dems.html#code-availability) section are outdated.

I guess the notebooks(_dem_comparison.ipynb_ and _dem_soures.ipynb_) to which these links are refering to are now located in the [oggm tutorials repository ](https://github.com/OGGM/tutorials/tree/master/notebooks).

**Comment by fmaussion** [comment]
Yes that's correct

**Comment by fmaussion** [comment]
Yes, you can correct this directly in the docs: https://github.com/GLIMS-RGI/rgitools/blob/master/docs/dems.rst - Let me know if you need help with specific urls
